docs/backend/database.md

```diff
@@ -43,10 +43,8 @@ password: password
 ### Other seed modes
 
 To put your local forem into "starter mode", as it would be for a new creator,
-use `MODE=STARTER` i.e...
+use `MODE=STARTER` skipping the creation of all sample data i.e.
```

Comment:
I thinks this and its below change cane be reverted as the current documentation looks correct here.